### PR TITLE
Adjust partner organization logos

### PR DESCRIPTION
**Pull request by gautamjajoo**
Continuation of #1888 .
@deshraj @RishabhJain2018

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@gautamjajoo Did you also resized the logos for the normal screen?

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Please review!

**Comment by deshraj** [comment]
Thanks, Gautam for the PR. Can you please do the following changes:

1. Increase the size of Facebook logo a little bit
2. Decrease the size of Mapillary logo to make it look consistent with others
3. Decrease the size of UMBC logo
4. Also, it would be nice if you can sort them by alphabetical order. 

Let me know if you have questions.

**Comment by gautamjajoo** [comment]
@deshraj Updated1 Here's the scrrenshot
capture

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -156,46 +156,46 @@ <h4><div>Who uses EvalAI?</div></h4>
...
+                                        <img src="dist/images/organizations/umbc.png" id="umbc-logo">
                                     </div>
                                </div>
                                <div class="col s12 l3">
                                     <div class="org-logo">
-                                        <img src="dist/images/organizations/ebay.png" id="ebay-logo">
+                                        <img src="dist/images/organizations/pitt.png" id="pitt-logo">
                                     </div>
-                               </div>
+                               </div>
```
Comment (deshraj) [inline frontend/src/views/web/landing.html]: Can you please remove the extra space after the `</div>`

**Comment by gautamjajoo** [comment]
@deshraj Also, I am facing the following issue using worker. I am not able to proceed on my task due to the error. Thanks!
screenshot-from-2018-11-20-23-24-35

**Comment by gautamjajoo** [comment]
@deshraj Please review!

**Comment by deshraj** [comment]
@gautamjajoo can you please horizontally align the Facebook's logo with other three?

**Comment by gautamjajoo** [comment]
@deshraj Done!
evalai_users

**Comment by deshraj** [comment]
This looks perfect. Thanks @gautamjajoo :)

**Pull request merged**